src/lib.rs

```diff
@@ -2,29 +2,32 @@ pub mod api;
 pub mod commons;
 pub mod contracts;
 pub mod db;
-pub mod defillama;
 pub mod http_client;
 pub mod ipfs;
 pub mod scanner;
 pub mod signer;
 pub mod specification;
 pub mod telemetry;
 
-use std::{env, process::exit, sync::Arc};
+use std::{env, num::NonZeroU32, process::exit, sync::Arc};
 
 use anyhow::Context;
 use diesel::{
     pg::PgConnection,
     r2d2::{ConnectionManager, Pool},
 };
+use governor::{Quota, RateLimiter};
 use tokio::task::JoinSet;
 use tracing::{info_span, Instrument};
 
-use crate::{commons::ChainExecutionContext, defillama::DefiLlamaClient, http_client::HttpClient};
+use crate::{commons::ChainExecutionContext, http_client::HttpClient};
 use diesel_migrations::{embed_migrations, EmbeddedMigrations, MigrationHarness};
 
 pub const MIGRATIONS: EmbeddedMigrations = embed_migrations!("./migrations");
 
+const MAX_CALLS_PER_SECOND_DEFILLAMA: u32 = 10;
```

Comment:
We should decrease this value a bit, given the 500 requests/min limit for DefiLlama APIs.

Reply:
Addressed in 6198090a6645e24d3588b724e1dd3f430605800a